pipelines/config/dependency-builds-cflinuxfs4-test.yml

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
did you mean to check-in this file, or was this just for temporary testing?

Reply:
I thought about leaving the pipeline for a few weeks to do a quick build of the dependencies without interfering with the main pipeline.

The way it is used is basically modifying the listed dependencies when starting work on a new buildpack, i.e. if you need to work with the Nodejs buildpack, the pipeline will only contain the dependencies of this one. These changes will not need to be committed, as this will be the only change (initial structure using .NetCore Buildpack as an example).

This test pipeline would exist until all dependencies are built for cflinuxfs4. After that, it can be deleted from Git and from Concourse.